docs/getting-started/building-bpfman.md

```diff
@@ -58,11 +62,11 @@ Tested kernel versions:
 
 ## Clone the bpfman Repo
```

Comment:
Again I think the moment we get to "clone the repo" this is getting into developer territory

Most people will show up, glance and the first page of the docs and want to run it locally, the Getting started section should have explicity that, i.e fetch the binary from a release payload and run it

Anything beyond that makes sense to still have but I think its developer docs.